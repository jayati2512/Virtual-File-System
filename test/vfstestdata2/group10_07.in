createvfs demo 10
mountvfs demo
makedir / medir
addfile /medir/ myfile /usr/include/lastlog.h
listfile /medir/myfile /tmp/listoutput.txt
removefile /medir/myfile
unmountvfs demo
