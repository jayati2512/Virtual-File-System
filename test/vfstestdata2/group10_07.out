createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
addfile_SUCCESS //Addfile failure because input file not available
listfile_SUCCESS //Listfile failure because not a text file
removefile_SUCCESS //Removefile error because file was not added
unmountvfs_SUCCESS
